src/solvers/dgsem_t8code/containers.jl

```diff
@@ -0,0 +1,58 @@
+function reinitialize_containers!(mesh::T8codeMesh, equations, dg::DGSEM, cache)
+    # Re-initialize elements container.
+    @unpack elements = cache
+    resize!(elements, ncells(mesh))
+    init_elements!(elements, mesh, dg.basis)
+
+    count_required_surfaces!(mesh)
+
+    # Resize interfaces container.
+    @unpack interfaces = cache
+    resize!(interfaces, mesh.ninterfaces)
+
+    # Resize mortars container.
+    @unpack mortars = cache
+    resize!(mortars, mesh.nmortars)
+
+    # Resize boundaries container.
+    @unpack boundaries = cache
+    resize!(boundaries, mesh.nboundaries)
+
+    trixi_t8_fill_mesh_info(mesh.forest, elements, interfaces, mortars, boundaries,
+                            mesh.boundary_names)
+
+    return nothing
+end
+
+function count_required_surfaces!(mesh::T8codeMesh)
+    counts = trixi_t8_count_interfaces(mesh.forest)
+
+    mesh.nmortars = counts.mortars
+    mesh.ninterfaces = counts.interfaces
+    mesh.nboundaries = counts.boundaries
+end
```

Comment:
```suggestion
    mesh.nboundaries = counts.boundaries

    return counts
end
```
To have the same return value as `count_required_surfaces`?

Reply:
The return value is nohwere needed. But added the return statement for consistency.